src/ruby/yast/yast.rb

```diff
@@ -183,7 +183,7 @@ def self.import(mname)
       if (stype == :function)
         m.module_eval <<-"END"
           def self.#{sname}(*args)
-            caller[0].match BACKTRACE_REGEXP
+            caller(1,1).first.match BACKTRACE_REGEXP
```

Comment:
and it was manually tested by installation with new ruby bindings. I expect it will be quickly recognized, if it is wrong.